pkg/cmd/version.go

```diff
@@ -159,15 +161,30 @@ func operatorInfo(ctx context.Context, c client.Client, namespace string) (map[s
 		}
 	}
 
+	catalog, err := camel.LoadCatalog(ctx, c, namespace, v1.RuntimeSpec{Version: platform.Status.Build.RuntimeVersion, Provider: platform.Status.Build.RuntimeProvider})
+	if catalog != nil {
+		for k, v := range catalog.CamelCatalogSpec.Runtime.Metadata {
+			infos[k] = v
+		}
+	} else {
+		log.Debug("Can't load CamelCatalog.")
```

Comment:
More than logging a debug trace, I think we should expose the error.

Reply:
I don't see a problem with that, on the contrary, I think that would be the correct behavior in particular for version 2, where a `CamelCatalog` is required.